.azure/pipelines/ci.yml

```diff
@@ -689,7 +689,6 @@ stages:
 
   - template: jobs/default-build.yml
     parameters:
-      condition: notin(variables['Build.Reason'], 'PullRequest')
```

Comment:
For testing only, I'll revert this before merging.

Reply:
My bad I should have reminded you to try running on all the other queues in the first PR too since I ran into the same thing when I turned on the templates test